cmd/convert.go

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func getBoolFlags() []string {
+	return []string{
+		"--debug", "-D",
+		"--verbose",
+		"--l",
+		"--tls",
+		"--tlsverivy",
+	}
+}
+
+func getStringFlags() []string {
+	return []string{
+		"--tlscacert",
+		"--tlscert",
+		"--tlskey",
+		"--host", "-H",
+		"--context",
+		"--log-level",
+	}
+}
+
+func convert(args []string) []string {
+	rootFlags := []string{}
```

Comment:
```suggestion
	var rootFlags []string
```
[`nil` is great](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices) 😁

Reply:
I also like. But I have people marking exactly the opposite on other PR's :D 
Nice to know I have a `nil`'s lover friend then!

Reply:
I never understood go let you use `nil` slices just like it gets magically lazy-allocated, but don't do the same with `map` 🤨

Reply:
Well, there is no `append` for maps :)

Reply:
Actually the allocation comes from the built-in `append` function

Reply:
> there is no append for maps :)

not being an explicit function doesn't mean go can't be clever. See `range` which supports nil maps for example. Could do something clever with